### PR TITLE
add changes to relente branch

### PR DESCRIPTION
**Pull request by relente-patriciajoy**
Please review my code.

### CODE REVIEW

File: bnpackage/relente.py [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+        match choice:
+            case 1:
+                print("I am PJ, a 2nd-year DIT Student from PUP-Taguig Campus.") 
+                print("I love petting my cat because it relieves my stress.")
+            case 2:
+                print("Goal: To have a happy and meaningful life")
+            case 3:
+                print("About my cat")
+                print("> Frico is a 2-year-old tuxedo cat")
+                print("> He loves Chimken.")
```
Comment (citron-kathleen) [inline bnpackage/relente.py]: Please check your code.

**Review by citron-kathleen — CHANGES_REQUESTED** [review]
Please check your spelling.

File: bnpackage/relente.py (hunk 1 above)
Comment (ynion-mabeamae) [inline bnpackage/relente.py]: Please remove the ">" and check your spelling.

Comment (delima-justine) [inline bnpackage/relente.py]: Change "chimken" to "chicken"

File: bnpackage/relente.py [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+            case 4:
+                return
+            case _:
+                print("Invalid choice. Please try again.")
+
+patricia_relente()
```
Comment (ynion-mabeamae) [inline bnpackage/relente.py:27]: Remove patricia_relente() and move to main.py

**Review by ynion-mabeamae — CHANGES_REQUESTED** [review]
Please check your code before I approved.

**Review by delima-justine — CHANGES_REQUESTED** [review]
Please review the code again based on the recommendations before I approve. Thank you!

**Review by quiambao-mapatriciaanne — CHANGES_REQUESTED** [review]
Fix your code before I approve it.

**Review by ynion-mabeamae — APPROVED** [review]
Good job!

**Review by delima-justine — APPROVED** [review]
Great!

**Review by citron-kathleen — APPROVED** [review]
Great job!

**Review by quiambao-mapatriciaanne — APPROVED** [review]
All done!

**Pull request merged**